kernel/src/common/regs/mod.rs

```diff
@@ -134,6 +134,11 @@ impl<T: IntLike, R: RegisterLongName> ReadWrite<T, R> {
         self.set((reg & !field.mask) | field.value);
     }
 
+    #[inline]
+    pub fn modify_no_read(&self, original: T, field: FieldValue<T, R>) {
```

Comment:
Why have this take in the original as a `T` rather than as a `LocalRegisterCopy<T, R>`?